ro-base.obo

```diff
@@ -613,6 +615,7 @@ property_value: IAO:0000116 "Location as a relation between instances: The primi
 property_value: IAO:0000116 "Most location relations will only hold at certain times, but this is difficult to specify in OWL. See http://purl.obolibrary.org/obo/ro/docs/temporal-semantics/" xsd:string
 property_value: IAO:0000118 "located_in" xsd:string
 property_value: RO:0001900 RO:0001901
+property_value: seeAlso Located:in
```

Comment:
Weird.

Reply:
Fixed in the commit [c0ecc1](https://github.com/oborel/obo-relations/pull/728/commits/c0ecc1df96a957bd979bc98214b85b855e6818ad)